tools/monorepo-utils/src/code-freeze/commands/branch/index.ts

```diff
@@ -20,10 +20,12 @@ import {
 } from '../../../core/github/repo';
 import { WPIncrement } from '../../../core/version';
 import { Logger } from '../../../core/logger';
-import { Options } from './types';
 import { getEnvVar } from '../../../core/environment';
 
-const getNextReleaseBranch = async ( options: Options ) => {
+const getNextReleaseBranch = async ( options: {
+	owner?: string;
+	repo?: string;
+} ) => {
```

Comment:
You've changed the variable from `name` to `repo`. I bet TS didn't pick this up as an error because of the `?`, so yeah good idea here. Probably best to change this back and tackle this in a follow up.

https://github.com/woocommerce/woocommerce/blob/52a268892b0323db30883d2b94597e33ce5d26a4/tools/monorepo-utils/src/core/github/repo.ts#L11-L14

So some of the graphql queries take `name` while the rest api uses `repo`. In development I went with `name` but I can see `repo` makes more sense. If we change here, it should be everywhere though

Reply:
Whoops, I'm super glad you spotted this. thank you.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/38185/commits/8fe2dd7d343217534784c61f8a79ee6224a7182c